frontend/src/components/NodeItem.tsx

```diff
@@ -282,18 +307,22 @@ const StyledLevel = styled(Typography)`
   font-family: inherit;
 `
 
-const StyledClaimButton = styled(Button)<{ isPlayerOwned: boolean }>`
+const StyledClaimButton = styled(Button)<{ isplayerowned: string }>`
   margin-left: auto;
   background: var(--color-black);
-  color: ${({isPlayerOwned}) => isPlayerOwned ? "var(--color-primary)" : "var(--color-secondary)"};
-  border: 2px solid ${({isPlayerOwned}) => isPlayerOwned ? "var(--color-primary)" : "var(--color-secondary)"};
+  color: ${({isplayerowned}) => isplayerowned === "true" ? "var(--color-primary)" : "var(--color-secondary)"};
+  border: 2px solid ${({isplayerowned}) => isplayerowned === "true" ? "var(--color-primary)" : "var(--color-secondary)"};
   border-radius: 8px;
   font: inherit;
   z-index: 5;
   
     &:active {
     background: var(--color-black);
     }
+  
+    &:hover {
+    background: var(--color-black);
+    }
```

Comment:
This hovereffect is not optimized for mobile, your app should be a mobile game aight?

Reply:
Here you can seperate behaviour by if the user has a cursor or not. The second media query addresses mobile users

```css
@media (hover: hover) {
    &:hover:enabled {
      background-color: var(--color-black);
    }
  }

  @media (hover: none) {
    &:active:enabled {
      background-color: var(--color-black);
    }
```

Reply:
Exactly, thanks for the media query. I actually just wanted to make sure the hover effect has no effect xD